packages/enzyme-example-mocha/src/Foo.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+class Foo extends React.Component {
+  constructor(props) {
+    super(props);
```

Comment:
is there any reason to keep this useless constructor?

Reply:
deleted. 
Also added eslint and nyc coverage and fixed up the eslint warnings